### ISSUE TITLE
Cannot put object acl without a policy

### ISSUE THREAD

**Issue by mkleint** [adobe/S3Mock#1617]
`aws --profile okta s3api put-object-acl --acl "bucket-owner-full-control" --key "data.txt" --bucket video-nonprod --endpoint-url=http://localhost:9090 --debug   
`
our code performs a java equivalent of the above. s3mock returns 400, I assume because the request body cannot be parsed, because we don't specify the policy

**Comment by afranken** [comment]
@mkleint does the code above work against S3?

The API clearly states:

```Request Body

The request accepts the following data in XML format.

[AccessControlPolicy](https://docs.aws.amazon.com/AmazonS3/latest/API/API_PutObjectAcl.html#API_PutObjectAcl_RequestSyntax)

    Root level tag for the AccessControlPolicy parameters.

    Required: Yes```

https://docs.aws.amazon.com/AmazonS3/latest/API/API_PutObjectAcl.html

So the request should fail on S3 as well.

**Comment by afranken** [comment]
ah!
looking at the API again, it seems like you're specifying a "canned ACL":
https://docs.aws.amazon.com/AmazonS3/latest/userguide/acl-overview.html#CannedACL
this is indeed currently not supported.

**Comment by mkleint** [comment]
The old code I'm migrating was using java aws client v2 with following code 
```
                                for (val o : s3ObjectList.contents()) {
                                    // update object ACL
                                    s3.putObjectAcl(PutObjectAclRequest.builder()
                                            .bucket(bucket)
                                            .key(o.key())
                                            .acl("bucket-owner-full-control")
                                            .build());
                                }
```
I assume it works, as we running that in production. unfortunately that old code is not using s3mock in CI to test.

the new code we creating now is doing the same and I've tried to add component test to verify in CI. I've tried to create the aws cli equivalent to test against the s3mock and to report as issue. No idea if the java client is adding some kind of dummy request body or not.

**Comment by afranken** [comment]
@mkleint I just released 3.5.1 that supports canned ACLs.

**Comment by mkleint** [comment]
great, thank you. works perfectly now
